src/expr/scalar/func.rs

```diff
@@ -2821,14 +2787,8 @@ impl fmt::Display for UnaryFunc {
             UnaryFunc::ExtractTimestampTzDayOfYear => f.write_str("tstzextractdayofyear"),
             UnaryFunc::ExtractTimestampTzDayOfWeek => f.write_str("tstzextractdayofweek"),
             UnaryFunc::ExtractTimestampTzIsoDayOfWeek => f.write_str("tstzextractisodayofweek"),
-            UnaryFunc::DateTruncTimestamp(to) => {
-                f.write_str("date_truncts_")?;
-                f.write_str(&format!("{:?}", to).to_lowercase())
-            }
-            UnaryFunc::DateTruncTimestampTz(to) => {
-                f.write_str("date_trunctstz_")?;
-                f.write_str(&format!("{:?}", to).to_lowercase())
-            }
+            UnaryFunc::DateTruncTimestamp(to) => write!(f, "date_trunc_{}_ts", to),
```

Comment:
I think you'll need to also edit the sqllogictests that assert on explain plan output?

Reply:
Yep, thanks!